ng2-components/ng2-alfresco-datatable/src/components/pagination/pagination.component.ts

```diff
@@ -20,7 +20,7 @@ import { Pagination } from 'alfresco-js-api';
 import { PaginationData } from '../../models/pagination.data';
 
 @Component({
-    selector: 'adf-pagination, alfresco-pagination',
+    selector: 'alfresco-pagination',
```

Comment:
Move both selectors to the core/pagination component and remove this implementation once backwards compatibility features are introduced.